api/v1/testkube.yaml

```diff
@@ -7488,6 +7489,9 @@ components:
           description: custom execution name
         config:
           $ref: "#/components/schemas/TestWorkflowConfigValue"
+        testWorkflowExecutionName:
```

Comment:
not important: maybe simpler to use just `executionName`?

Reply:
or maybe something that is more related with CR, considering that it's not changing the execution name?

Reply:
yes, the idea was to have it similar to TestWorkflowExecution.Name